<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComDialog</class>
 <widget class="QDialog" name="ComDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>关于</string>
  </property>
  <property name="windowIcon">
   <iconset resource="hello.qrc">
    <normaloff>:/logo/Resources/fivision.ico</normaloff>:/logo/Resources/fivision.ico</iconset>
  </property>
  <widget class="QLineEdit" name="lineEdit_OtherInfo">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="QPushButton_OpenComm">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>510</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>打开串口</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Y0">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>110</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Y0</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="QPushButton_Send_SencondQueue">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>50</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Slave">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_FunctionCode">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_StartAddress">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Slave</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Function Code</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Address</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Other Info</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_LRC">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>60</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>LRC</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_DefaultQueue">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>421</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="QPushButton_Send_DefaultQueue">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>150</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_TimeLag">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>370</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="QPushButton_TimeLag">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>360</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>间隔时间(ms)</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_Slave</tabstop>
  <tabstop>lineEdit_FunctionCode</tabstop>
  <tabstop>lineEdit_StartAddress</tabstop>
  <tabstop>lineEdit_OtherInfo</tabstop>
  <tabstop>QPushButton_Send_SencondQueue</tabstop>
  <tabstop>QPushButton_OpenComm</tabstop>
  <tabstop>checkBox_Y0</tabstop>
  <tabstop>lineEdit_LRC</tabstop>
 </tabstops>
 <resources>
  <include location="hello.qrc"/>
 </resources>
 <connections/>
</ui>
